mail-client/mailspring-bin/mailspring-bin-1.8.0.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit unpacker
+
+DESCRIPTION="A beautiful, fast and fully open source mail client for Mac, Windows and Linux."
+
+HOMEPAGE="https://getmailspring.com/"
+
+SRC_URI="https://github.com/Foundry376/Mailspring/releases/download/${PV}/mailspring-${PV}-amd64.deb"
+
+LICENSE="GPL-3"
+
+SLOT="0"
+
+KEYWORDS="~amd64"
+
+IUSE=""
```

Comment:
Remove it, see https://devmanual.gentoo.org/ebuild-writing/variables/index.html#iuse

Reply:
Thanks!